packages/harpguru-core/src/components/harp-cell/harp-cell.tsx

```diff
@@ -50,24 +51,35 @@ export const HarpCell = ({ yxCoord }: HarpCellProps): React.ReactElement => {
   const handleTapStateChange = ({
     nativeEvent,
   }: TapGestureHandlerStateChangeEvent) => {
-    if (nativeEvent.state !== State.END) return
-
-    addBufferedActivityToggle(thisDegreeId)
+    const interestingTouchStates = [State.BEGAN, State.CANCELLED, State.FAILED]
+    if (interestingTouchStates.includes(nativeEvent.state)) {
+      setIsTouchState(nativeEvent.state)
+    }
+    if (nativeEvent.state === State.END) {
+      addBufferedActivityToggle(thisDegreeId)
+    }
   }
 
+  React.useEffect(() => {
+    if (touchState !== State.UNDETERMINED) {
+      setIsTouchState(State.UNDETERMINED)
+    }
+  }, [thisIsActiveId, setIsTouchState])
+
   const harpCellAccessibleProps = {
     degreeId: thisDegreeId,
     pitchId: thisPitchId,
     isActive: thisIsActiveId === IsActiveIds.Active,
     displayMode: activeDisplayMode,
     activeExperienceMode: activeExperienceMode,
     baseStyles: baseHarpCellStyles,
+    touchState,
   }
 
   return (
     <LongPressGestureHandler
       onHandlerStateChange={handleLongPressStateChange}
-      minDurationMs={500}
+      minDurationMs={5000}
```

Comment:
This was only set so high to clear the  way for playing with the cells. It should be set back at some point.

Reply:
It's been set to 1 second which I think I prefer now